lms/djangoapps/ccx/views.py

```diff
@@ -80,6 +80,7 @@ def wrapper(request, course_id):
         """
         course_key = CourseKey.from_string(course_id)
         ccx = None
+        is_course_licensing_enable = run_extension_point('PCO_ENABLE_COURSE_LICENSING')
```

Comment:
```suggestion
        is_course_licensing_enabled = run_extension_point('PCO_ENABLE_COURSE_LICENSING')
```

Do you think it would be worth the effort to change the name of the variable in all definitions on all files?